src/client/components/search-button/index.js

```diff
@@ -0,0 +1,15 @@
+import { h, Component } from "preact";
+import cx from "classnames";
+
+import "./style.scss";
+
+const SearchButton = ({ google, stackoverflow, href = "#" }) => {
+  let classes = cx(
```

Comment:
Pshh, use `obj-str` 😜 

```js
import cx from 'obj-str';

let cls = cx({ superlink:true, google, stackoverflow });
```

Reply:
reflecting this -- look after your friends

Reply:
If you needed the array syntax, I wouldn't suggest it